imap_processing/hit/l1a/hit_l1a.py

```diff
@@ -50,203 +47,207 @@ def hit_l1a(packet_file: str, data_version: str) -> list[xr.Dataset]:
 
     Returns
     -------
-    cdf_filepaths : list[xarray.Dataset]
+    processed_data : list[xarray.Dataset]
         List of Datasets of L1A processed data.
     """
-    # Decom, sort, and group packets by apid
-    packets = decom_packets(packet_file)
-    sorted_packets = utils.sort_by_time(packets, "SHCOARSE")
-    grouped_data = group_data(sorted_packets)
-
-    # create the attribute manager for this data level
+    # TODO add logging
+
+    # Unpack ccsds file
+    packet_definition = (
+        imap_module_directory / "hit/packet_definitions/" "hit_packet_definitions.xml"
+    )
+    datasets_by_apid = packet_file_to_datasets(
```

Comment:
the `use_dervied_values` flag needs to be set to `False` for L1A

Reply:
:( Yeah, I should just set this as the default already!